systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PersistentVolumeClaimUtils.java

```diff
@@ -89,8 +90,8 @@ public static void waitForPersistentVolumeClaimDeletion(TestStorage testStorage,
         );
     }
 
-    public static void waitForJbodStorageDeletion(String namespaceName, int volumesCount, JbodStorage jbodStorage, String clusterName) {
-        int numberOfPVCWhichShouldBeDeleted = jbodStorage.getVolumes().stream().filter(
+    public static void waitForJbodStorageDeletion(String namespaceName, int volumesCount, String clusterName, SingleVolumeStorage... volumes) {
```

Comment:
Maybe you can use `List` here and in the place where you are using this method pass the parameters to List using `List.of()`, but I guess it is just a matter of preference

Reply:
thanks, will do this in next PR (also touching this test).

Reply:
https://github.com/strimzi/strimzi-kafka-operator/pull/8461